src/components/rangeslider/attributes.js

```diff
@@ -30,6 +30,22 @@ module.exports = {
         role: 'style',
         description: 'Sets the border color of the range slider.'
     },
+    range: {
+        valType: 'info_array',
+        role: 'info',
+        items: [
+            {valType: 'number'},
+            {valType: 'number'}
+        ],
+        description: [
+            'Sets the initially selected range for the range slider.',
```

Comment:
@mdtusz Hmm. That's not how I imagined it.

I was thinking that the *initially selected range* would be given by `xaxis.range` - as done currently on first render as well as on range updates.

Instead, `xaxis.rangeslider.range` would correspond to the end-to-end range (or span) of the range slider.

For example,

```js
layout.xaxis = {
  range: [10, 20],
  rangeslider: {
    range: [0, 30]
  }
}
```

 would imply the the initially selected range is `[10, 20]` whereas the slider has a span of `[0, 30]`

![image](https://cloud.githubusercontent.com/assets/6675409/14746743/0f7b3b16-087f-11e6-8370-f2e967eeb1be.png)